astropy/units/si.py

```diff
@@ -63,6 +63,18 @@
 )
 
 
+###########################################################################
+# AREA
+
+def_unit(
+    ["ha", "hectare"], # CAVE: ha could be confused with hecto-annum
+    1e4 * m**2,
+    namespace=_ns,
+    prefixes=False,
+    doc="hectare: unit of area. Is used to express land area",
```

Comment:
Nitpick, but maybe better as "unit of area, used to express area of land"

Reply:
done as suggested